### ISSUE TITLE
Suggest adding more examples to 3.5.1 User Story: Distractions

### ISSUE THREAD

**Issue by julierawe** [w3c/coga#164]
"I do not want distractions from my task."
Suggest adding examples of distractions, such as:
- Multiple links
- Pop-up windows

Also, in the 1st sentence of this section, suggest rewording to remove judgmental term ("weak"; see issue #140):
Change "As a user with an attention impairment and weak memory," to "As a user with attention and memory impairments,"

**Comment by jpascalides** [comment]
Thank you for your suggestions. We have removed the word "weak" from the text. Unfortunately, the group agreed to avoid adding the examples to the bulleted list.
